src/tabs/pid_tuning.html

```diff
@@ -762,7 +762,7 @@
                             </tr>
 
                             <tr class="idleMinRpm">
-                                <td><input type="number" name="idleMinRpm-number" step="1" min="0" max="100"/></td>
+                                <td><input type="number" name="idleMinRpm-number" step="1" min="0" max="200"/></td>
```

Comment:
This does not depend of the Betaflight version?

Reply:
Thanks. Updated. As the PG version in firmware has not been raised we should keep it designate this for 4.4 anyway?

Reply:
In this case the CLI command `dyn_idle_min_rpm` is backwards compatible cause we're only extending the range, not the underlying data type. So we could add it to 4.3.2.

Reply:
Thanks. Changed to 4.3